bip-update-process.md

```diff
@@ -0,0 +1,706 @@
+The MAJOR version is incremented if changes to the BIP’s Specification are introduced that are incompatible with prior
+versions (which should be rare after a BIP is Complete, and only happen in well-grounded exceptional cases to a BIP that
+is Deployed). The MINOR version is incremented whenever the specification of the BIP is changed or extended in a
+backward-compatible way. The PATCH version is incremented for other changes to the BIP that are noteworthy (bug fixes,
+test vectors, important clarifications, etc.). Version 1.0.0 is used to label the promotion to
+Complete. A Changelog section may be introduced during the Draft phase to record significant changes (using versions 0.x.y). An example of a Changelog section can be seen in
+[BIP 352](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki). After a BIP receives a Changelog, the
+Preamble must indicate the latest version in the Revision header.
+
+### Adoption of proposals
+
+The BIP repository does not track the sentiment on proposals and does not track the adoption of BIPs beyond whether they
+are in active use or not. It is not intended for BIPs to list additional implementations beyond the reference
+implementation: the BIP repository is not a signpost where to find implementations.[^OtherImplementations] After a BIP
+is advanced to Complete, it is up to the Bitcoin community to evaluate, adopt, ignore, or reject a BIP. Individual
+Bitcoin projects are encouraged to publish a list of BIPs they implement. A good example of this at the time of writing
+this BIP can be observed in Bitcoin Core’s [doc/bips.md](https://github.com/bitcoin/bitcoin/blob/master/doc/bips.md)
+file.
+
+### Transferring BIP Ownership
+
+It occasionally becomes necessary to transfer ownership of BIPs to new owners. In general, it would be preferable to
+retain the original authors of the transferred BIP, but that is up to the original authors. A good reason to transfer
+ownership is because the original authors no longer have the time or interest in updating it or following through with
+the BIP process, or have fallen off the face of the 'net (i.e., are unreachable or not responding to email). A bad reason
+to transfer ownership is because someone doesn't agree with the direction of the BIP. The community tries to build
+consensus around a BIP, but if that's not possible, rather than fighting over control, the dissenters should supply a
+competing BIP.
+
+If someone is interested in assuming ownership of a BIP, they should send an email asking to take over, addressed to the
+original authors, the BIP Editors, and the Bitcoin Development Mailing List. If the authors do not respond in a timely
+manner (e.g., two weeks), the BIP editors will make a unilateral decision whether to appoint the applicants as
+[Shepherds](#authors-and-shepherds) (which may be amended should the original authors make a delayed reappearance).
```

Comment:
If the BIP is in draft, a new owner may be an author as well? (It would be simpler to have a single role)